src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -65,6 +53,28 @@ class FileList extends React.Component {
     }
   }
 
+  ensureActiveItemVisible() {
+    if (!this.activeFileItem) {
+      return;
+    }
+    computeScrollIntoView(this.activeFileItem, { scrollMode: 'if-needed' }).forEach((action) => {
+      if (action.el.className !== 'dam-file-list') { // fixme: use the class name from a ref instead of hardcoding
```

Comment:
@digitalartcrew do the todo...make this a ref instead of hardcoding the class name